public/contribute.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "portal-alpha",
+  "description": "File Sharing Experiment",
+  "repository": {
+    "url": "https://github.com/mozilla/something-awesome/",
+    "license": "MPL-2.0"
+  },
+    "participate": {
+      "home": "https://github.com/mozilla/something-awesome/blob/master/README.md",
```

Comment:
Not sure if we have a good Wiki landing page for the `participate.home` field either.